CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 - Fixed a bug that causes output of GroupBy.aggregate's columns to be ordered incorrectly.
 - Fixed a bug where `DataFrame.describe` on a frame with duplicate columns of differing dtypes could cause an error or incorrect results.
 - Fixed a bug in `DataFrame.rolling` and `Series.rolling` so `window=0` now throws `NotImplementedError` instead of `ValueError`
+- Fixes a bug in `DataFrame` and `Series` with `dtype=np.uint64` resulting in precision errors
```

Comment:
This should be a fix for python API too.

Reply:
Spoke with Afroz, we are going to bundle this change in with other BCR changes that don't affect APIs, JIRA here: https://snowflakecomputing.atlassian.net/browse/BCR-1694